site/src/pages/ManagementSettingsPage/OrganizationProvisionersPageView.stories.tsx

```diff
@@ -53,21 +58,50 @@ export const Provisioners: Story = {
 				],
 			},
 			{
-				key: { ...MockProvisionerKey, id: "ケイラ", name: "ケイラ" },
-				daemons: [
-					{
-						...MockProvisioner,
-						tags: {
-							...MockProvisioner.tags,
-							都市: "ユタ",
-							きっぷ: "yes",
-							ちいさい: "no",
-						},
+				key: {
+					...MockProvisionerKey,
+					id: "ケイラ",
+					name: "ケイラ",
+					tags: {
+						...MockProvisioner.tags,
+						都市: "ユタ",
+						きっぷ: "yes",
+						ちいさい: "no",
 					},
-				],
+				},
+				daemons: Array.from({ length: 117 }, (_, i) => ({
+					...MockProvisioner,
+					id: `ケイラ-${i}`,
+					name: `ケイラ-${i}`,
+				})),
 			},
 		],
 	},
+	play: async ({ step }) => {
+		await step("open all details", async () => {
+			const expandButtons = await screen.findAllByText(
+				"Show provisioner details",
+			);
```

Comment:
Could this be updated to be more accessibility-friendly? I'm just assuming Storybook has a similar API to Testing Library:
```suggestion
const expandedButtons = await screen.findAllByRole("button", {
  name: "Show provisioner details"
});
```